extensions/src/ACRE2TS/TS3Client.cpp

```diff
@@ -199,6 +201,7 @@ acre::Result CTS3Client::localStopSpeaking(const acre::Speaking speakingType_) {
         case acre::Speaking::direct:
             break;
         case acre::Speaking::god:
+        case acre::Speaking::zeus:
```

Comment:
This is a C++17 feature indicating that you intentionally let a case fall through. 
```suggestion
            [[fallthrough]]
        case acre::Speaking::zeus:
```